File: src/main/java/org/kokotosi/scheduler/SchedulerLibrary.java
// SPDX-License-Identifier: Apache-2.0

package org.kokotosi.scheduler;

public final class SchedulerLibrary {
    public static final String NAME = "scheduler";
    public static final String STATUS = "bootstrap";

    private SchedulerLibrary() {}

    public static String banner() {
        return NAME + ":" + STATUS;
    }
}

File: src/main/kotlin/org/kokotosi/scheduler/SchedulerLibrary.kt
// SPDX-License-Identifier: Apache-2.0

package org.kokotosi.scheduler

/**
 * Temporary library entry point while the execution model is still being designed.
 * The point of this type is to keep the repository library-first from the first commit.
 */
object SchedulerLibrary {
    const val NAME: String = "scheduler"
    const val STATUS: String = "bootstrap"

    fun banner(): String = "$NAME:$STATUS"
}
